### PR TITLE
Update uk_1990.json

### PR DESCRIPTION
**Pull request by ColonelAkirNakesh**
Replaced USS Tarawa with HMS Invincible. 

Successfully tested replacing Tarawa w Invincible as valid control point on Pacific Repartee with four Harriers parked.

### CODE REVIEW

File: resources/factions/uk_1990.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "FFG Oliver Hazard Perry",
     "DDG Arleigh Burke IIa",
     "CG Ticonderoga",
-    "LHA-1 Tarawa",
+    "R05 Invincible"
```
Comment (MetalStormGhost) [inline resources/factions/uk_1990.json]: Comma missing from here?

Comment (ColonelAkirNakesh) [reply resources/factions/uk_1990.json]: Thanks, yeah! I removed the Stennis when I tested it but then re-added it but forgot the comma... doh!

Comment (MetalStormGhost) [reply resources/factions/uk_1990.json]: Please add the comma, otherwise we can't merge this since it wouldn't be valid json.

Comment (ColonelAkirNakesh) [reply resources/factions/uk_1990.json]: D'oh, thanks, fixed. Sorry for being such a nub.

**Pull request merged**